File: CS50's X/WEEK 6/sentimental-mario-more/mario.py
from cs50 import get_int


def main():

    height = get_height()

    for i in range(height):
        print_row(height - i - 1, i + 1)


def get_height():

    while True:
        height = get_int("Height: ")
        if height > 0 and height < 9:
            return height


def print_row(spaces, briks):

    for i in range(spaces):
        print(" ", end="")

    for i in range(briks):
        print("#", end="")

    print("  ", end="")

    for i in range(briks):
        print("#", end="")

    print()


main()

File: CS50's X/WEEK 1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

void print_row(int spaces, int briks);

int main(void)
{
    int height;
    do0
    {
        height = get_int("Height: ");
    }
    while (height < 1);

    for (int i = 0; i < height; i++)
    {
        print_row(height - i - 1, i + 1);
        printf("\n");
    }
}

void print_row(int spaces, int briks)
{
    for (int i = 0; i < spaces; i++)
        printf(" ");

    for (int i = 0; i < briks; i++)
        printf("#");
}
